.github/workflows/auto-author-assign.yml

```
@@ -0,0 +1,14 @@
+name: Auto Author Assign
```

Following the pattern of our existing workflows,

```suggestion
name: auto-author-assign
```